ray-operator/config/samples/ray-service.autoscaler.yaml

```diff
@@ -111,5 +95,5 @@ spec:
                     cpu: "1"
                     memory: "2Gi"
                   requests:
-                    cpu: "1"
+                    cpu: "500m"
```

Comment:
It is better to keep the resource request and limit the same. In my understanding, KubeRay will use the container's resource limits to set `rayStartParams`.

Reply:
Changed back!